cmd/clusterctl/examples/vsphere/generate-yaml.sh

```diff
@@ -98,14 +98,21 @@ mkdir -p ${OUTPUT_DIR}
 
 # all variables used for yaml generation
 
-CLUSTER_NAME=${CLUSTER_NAME:-vsphere-cluster}
-SERVICE_CIDR=${SERVICE_CIDR:-100.64.0.0/13}
-CLUSTER_CIDR=${CLUSTER_CIDR:-100.96.0.0/11}
-
-VSPHERE_USER=${VSPHERE_USER:-}
-VSPHERE_PASSWORD=${VSPHERE_PASSWORD:-}
-VSPHERE_SERVER=${VSPHERE_SERVER:-}
-VSPHERE_MANAGER_IMG=${VSPHERE_MANAGER_IMG:-}
+export CLUSTER_NAME=${CLUSTER_NAME:-vsphere-cluster}
+export SERVICE_CIDR=${SERVICE_CIDR:-100.64.0.0/13}
```

Comment:
Hi @andrewsykim,

Perhaps make a note that these CIDRs must match those used by whatever CNI add-on is installed?

Reply:
This is only the bootstrap cluster and we automatically install calico there with the same CIDRs injected

Reply:
Hi @andrewsykim,

Only if some other CNI plug-in were to come along that is better than Calico in every way that we would want to use. Do you feel there's a chance that could happen?